openpype/plugins/load/delivery.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, contexts, log=None, parent=None):
         template_label.setCursor(QtGui.QCursor(QtCore.Qt.IBeamCursor))
         template_label.setTextInteractionFlags(QtCore.Qt.TextSelectableByMouse)
 
+        frame_offset_value = QtWidgets.QSpinBox()
+        frame_offset_value.setMinimum(-99)
```

Comment:
I think this must be unlimited number - for example I might want to shift render from 1001 to 0001

Reply:
yep, that would be really useful

Reply:
> I feel that code suggestion should _definitely_ come with a comment to explain what it does. Would using [`sys.maxint`](https://docs.python.org/2/library/sys.html#sys.maxint) or alike work and be more readable? :)

oh, just saw your comment. `sys.maxint` didn't make it to python 3 and this is even more bizzare because this type size depends more on Qt binaries than Python. So this is just attempt to get 32bit size integer number divided by two to get maximum signed integer number, even though it might be totally off because you can build PySide2 with different sizes.